### PR TITLE
Fix Lint error

### PR DESCRIPTION
**Pull request by droibit**

### CODE REVIEW

File: app/src/main/java/com/github/android/dangerandroidsample/SampleApplication.kt [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+import android.app.Application
+import timber.log.Timber
+
+class SampleApplication : Application() {
+
+  override fun onCreate() {
+    super.onCreate()
+
+    if (BuildConfig.DEBUG){
```
Comment (droibit-sensei) [inline app/src/main/java/com/github/android/dangerandroidsample/SampleApplication.kt]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/droibit/danger-android-sample/blob/9695fd0e52135600b4d6d3f13bcd1a956238c991/app/src/main/java/com/github/android/dangerandroidsample/SampleApplication.kt#L11"></span>Missing spacing before “{“</td>
    </tr>
  </tbody>
</table>

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1,3 +1,5 @@
 <resources>
   <string name="app_name">Danger Android Sample</string>
+
+  <string name="unused_text">text</string>
```
Comment (droibit-sensei) [inline app/src/main/res/values/strings.xml]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/droibit/danger-android-sample/blob/9695fd0e52135600b4d6d3f13bcd1a956238c991/app/src/main/res/values/strings.xml#L4"></span>The resource <code>R.string.unused_text</code> appears to be unused</td>
    </tr>
  </tbody>
</table>

Comment (droibit) [reply app/src/main/res/values/strings.xml]: I fixed.

File: app/src/main/java/com/github/android/dangerandroidsample/SampleApplication.kt (hunk 1 above)
Comment (droibit) [reply app/src/main/java/com/github/android/dangerandroidsample/SampleApplication.kt]: I fixed

**Pull request merged**